.github/workflows/welcome-new-lgtm.yml

```diff
@@ -0,0 +1,19 @@
+name: Github - Welcome First-timers
+on:
+  pull_request:
+    types: [opened, closed]
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: wow-actions/welcome@v1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          FIRST_PR_MERGED: |
+            #### :sparkles: :tada: **AWESOME!** :tada: :sparkles:
+
+            Hi @{{ author }},
```

Comment:
I am just waiting for the day this fails with:

> Hi undefined,

Reply:
Will it? I am not sure how to accurately test this. I used the template they gave us.

Reply:
🤣🤣

Reply:
Ah - I see what you did here Shaun.

Reply:
> Will it?

Only if any of this API is written in JS. Otherwise, it is just a wish of mine.

Reply:
Goodbye Shaun. 😡